src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -962,6 +962,51 @@ def review_privacy_request(
     )
 
 
+@router.post(
+    PRIVACY_REQUEST_VERIFY_IDENTITY,
+    status_code=HTTP_200_OK,
+    response_model=PrivacyRequestResponse,
+)
+def verify_identification_code(
+    privacy_request_id: str,
+    *,
+    db: Session = Depends(deps.get_db),
+    provided_code: SubjectIdentityVerificationBodyParams,
```

Comment:
ah, I now realize `SubjectIdentityVerificationBodyParams` was badly named. That was anticipated to only account for email body params, not a request for subject identity verification. Could we make a new class to account for the request body here?

Reply:
sure can, an earlier commit already has it!
https://github.com/ethyca/fidesops/pull/1111/commits/40fcf6d119135d08a6d3ecfc40c5d73846bf2205